include/deal.II/matrix_free/matrix_free.h

```diff
@@ -242,6 +242,30 @@ class MatrixFree : public Subscriptor
           cell_vectorization_categories_strict)
     {}
 
+    /**
+     * Copy constructor.
+     */
+    AdditionalData(const AdditionalData &other)
+      : tasks_parallel_scheme(other.tasks_parallel_scheme)
+      , tasks_block_size(other.tasks_block_size)
+      , mapping_update_flags(other.mapping_update_flags)
+      , mapping_update_flags_boundary_faces(
+          other.mapping_update_flags_boundary_faces)
+      , mapping_update_flags_inner_faces(other.mapping_update_flags_inner_faces)
+      , mapping_update_flags_faces_by_cells(
+          other.mapping_update_flags_faces_by_cells)
+      , mg_level(other.mg_level)
+      , level_mg_handler(this->mg_level)
+      , store_plain_indices(other.store_plain_indices)
+      , initialize_indices(other.initialize_indices)
+      , initialize_mapping(other.initialize_mapping)
+      , overlap_communication_computation(
+          other.overlap_communication_computation)
+      , hold_all_faces_to_owned_cells(other.hold_all_faces_to_owned_cells)
```

Comment:
Did you forget `cell_vectorization_category`?

Reply:
Fixed.